packages/components/src/internal/components/lineage/utils.ts

```diff
@@ -9,6 +9,7 @@ import { imageURL, SchemaQuery, SCHEMAS, Theme } from '../../..';
 
 import { LineageItemWithMetadata, LineageLink, LineageNode } from './models';
 import { LINEAGE_DIRECTIONS, LineageIconMetadata } from './types';
+import React, { ReactNode } from 'react';
```

Comment:
was this needed? maybe it was added and then not cleaned up after some code was moved?

Reply:
Yes.  Removed now.